pkg/reconciler/clusteringress/resources/secret.go

```diff
@@ -93,3 +93,8 @@ func SecretRef(namespace, name string) corev1.ObjectReference {
 		Name:       name,
 	}
 }
+
+// Generates the k8s secret key with the given TLS.
+func secretKey(tls *v1alpha1.ClusterIngressTLS) string {
```

Comment:
nit: why does this need a pointer?

Reply:
It does not have to be a pointer as we don't change the `tls` content in this function.